osu.Game/Localisation/MouseSettingsStrings.cs

```diff
@@ -15,19 +15,19 @@ public static class MouseSettingsStrings
         public static LocalisableString Mouse => new TranslatableString(getKey(@"mouse"), @"Mouse");
 
         /// <summary>
-        /// "Not applicable in full screen mode"
+        /// "not applicable in full screen mode"
         /// </summary>
-        public static LocalisableString NotApplicableFullscreen => new TranslatableString(getKey(@"not_applicable_full_screen"), @"Not applicable in full screen mode");
+        public static LocalisableString NotApplicableFullscreen => new TranslatableString(getKey(@"not_applicable_full_screen"), @"not applicable in full screen mode");
 
         /// <summary>
         /// "High precision mouse"
         /// </summary>
         public static LocalisableString HighPrecisionMouse => new TranslatableString(getKey(@"high_precision_mouse"), @"High precision mouse");
 
         /// <summary>
-        /// "Attempts to bypass any operation system mouse acceleration. On windows, this is equivalent to what used to be known as &quot;Raw Input&quot;."
+        /// "attempts to bypass any operating system mouse acceleration. on windows, this is equivalent to what used to be known as &quot;raw input&quot;."
         /// </summary>
-        public static LocalisableString HighPrecisionMouseTooltip => new TranslatableString(getKey(@"high_precision_mouse_tooltip"), @"Attempts to bypass any operation system mouse acceleration. On windows, this is equivalent to what used to be known as ""Raw Input"".");
+        public static LocalisableString HighPrecisionMouseTooltip => new TranslatableString(getKey(@"high_precision_mouse_tooltip"), @"attempts to bypass any operating system mouse acceleration. on windows, this is equivalent to what used to be known as ""raw input"".");
```

Comment:
```suggestion
        public static LocalisableString HighPrecisionMouseTooltip => new TranslatableString(getKey(@"high_precision_mouse_tooltip"), @"attempts to bypass any operating system mouse acceleration. on Windows, this is equivalent to what used to be known as ""raw input"".");
```
What about proper nouns?